docs/mappings/cambio_gbfs_2.3_mapping.md

```diff
@@ -0,0 +1,258 @@
+
+# Cambio to GBFS Mapping
+
+This document maps the Cambio API to GBFS.
+
+# Reference version
+
+This documentation refers to **[GBFS v2.3](https://github.com/MobilityData/gbfs/blob/v2.3/gbfs.md)**.
+
+## Table of Contents
+
+* [Introduction](#introduction)
+* [General Information](#general-information)
+* [Open Issues or Questions](#open-issues-or-questions)
+* [Files](#files)
+    * [gbfs.json](#gbfsjson)
+    * [gbfs_versions.json](#gbfs_versionsjson) *(added in v1.1)*
+    * [system_information.json](#system_informationjson)
+    * [vehicle_types.json](#vehicle_typesjson) *(added in v2.1)*
+    * [station_information.json](#station_informationjson)
+    * [station_status.json](#station_statusjson)
+    * [free_bike_status.json](#free_bike_statusjson)
+    * [system_hours.json](#system_hoursjson)
+    * [system_calendar.json](#system_calendarjson)
+    * [system_regions.json](#system_regionsjson)
+    * [system_pricing_plans.json](#system_pricing_plansjson)
+    * [system_alerts.json](#system_alertsjson)
+    * [geofencing_zones.json](#geofencing_zonesjson) *(added in v2.1)*
+
+## Introduction
+Cambio only provides static information, so GBFS feeds generated for Cambio don't provide realtime information. The following sections describe how Cambio's API is mapped to GBFS.
+
+## General Information
+
+Cambio provides static endpoints per city to retrieve their stations and vehicles, &lt;city&gt; to be replaced by the city short name, e.g. AAC for Aachen:
+
+* https://cwapi.cambio-carsharing.com/opendata/v1/mandator/&lt;city&gt;/stations
+* https://cwapi.cambio-carsharing.com/opendata/v1/mandator/&lt;city&gt;/vehicles
+
+
+
+### Example
+
+The following fragment is an example of the Cambio station document:
+
+```json
+[ {
+  "id" : "15-4568",
+  "displayName" : "Dom",
+  "name" : "Dom",
+  "address" : {
+    "streetAddress" : "Jesuitenstr.",
+    "streetNumber" : "12",
+    "addressLocation" : "Aachen",
+    "postalCode" : "52062",
+    "addressCountry" : "DE"
+  },
+  "information" : {
+    "description" : "Karte zum Ã–ffnen der Schranken bei Ein- und Ausfahrt befindet sich am SchlÃ¼sselbund. Karte direkt vor das Lesefeld halten. Lesefeld am jeweiligen Schrankenautomaten rechts unten ausgewiesen. Karte mindestens eine Sekunde vorhalten.\n\nBeim Verlassen des Stellplatzes unbedingt den rotweiÃŸen Poller aufrichten. Der Dreikant-SchlÃ¼ssel zum Entriegeln und Niederlegen des Pollers liegt im Fahrzeug (Handschuhfach oder TÃ¼rablage).",
```

Comment:
This seems to be ISO-8859, interpreted as UTF-8.

Reply:
Indeed, fixed with 07fd00d04909670d62dff336367f72804bafaf3d